SimTKcommon/include/SimTKcommon/internal/Value.h

```diff
@@ -9,7 +9,7 @@
  * Biological Structures at Stanford, funded under the NIH Roadmap for        *
  * Medical Research, grant U54 GM072970. See https://simtk.org/home/simbody.  *
  *                                                                            *
- * Portions copyright (c) 2005-12 Stanford University and the Authors.        *
+ * Portions copyright (c) 2005-15 Stanford University and the Authors.        *
  * Authors: Michael Sherman                                                   *
  * Contributors:                                                              *
```

Comment:
Add yourself to the contributors list, unless you prefer anonymity. People usual add themselves to Contributors for small changes or to Authors for major changes.